tests/unit/test_file_operations.py

```diff
@@ -57,6 +62,11 @@ def test_log_operation(self):
             content = f.read()
         self.assertIn(f"log_test: {self.test_file}", content)
 
+        log_operation("log_test2", self.test_file, "test")
+        with open(self.config.file_logger_path, "r") as f:
+            content = f.read()
+        self.assertIn(f"log_test2: {self.test_file}", content)
+
```

Comment:
out of scope

Reply:
Not out of scope as these are tests are applicable on the required changes in file_operations.py, as mentioned above